.github/workflows/slot_machine.yml

```diff
@@ -0,0 +1,36 @@
+name: Slot Maschine
```

Comment:
@cbisaillon Actions might be interesting for others. If it's unwanted then I will delete it from this branch.

Reply:
Can you remove this please ?